styles/pdp.css

```diff
@@ -0,0 +1,153 @@
+.search_navbar {
+  padding-left: 10px;
+  width: var(--width-100);
+}
+.cake-logo {
+  color: var(--color-white);
+  font-size: var(--size-7);
+}
+.fa-search {
+  position: absolute;
+  top: 43%;
+  left: 24%;
+  cursor: pointer;
+}
+.list {
+  justify-content: space-around;
+}
+.search_navbar {
+  padding: 10px;
+  width: var(--width-100);
+}
+.cake-logo {
+  color: var(--color-white);
+  font-size: var(--size-7);
+}
+.fa-search {
+  position: absolute;
+  top: 43%;
+  left: 24%;
+  cursor: pointer;
+}
+.list {
+  justify-content: space-around;
+}
+.heading-title {
+  color: var(--color-white);
+  font-size: var(--size-5);
+}
+.logo {
+  display: flex;
+  gap: var(--size-2);
+}
+.cart-container {
+  display: flex;
+  gap: var(--size-2);
+  width: 100%;
+  overflow: visible;
+  align-items: center;
+  justify-content: center;
+}
+
+.main-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  margin-top: 10%;
```

Comment:
You can remove this line. Margin top is getting applied in the next line anyway.

Reply:
yeah have removed it thanks for pointing out.